packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -323,16 +328,10 @@ function CropCorner({
 }): VNode {
   return (
     <button
-      class="crop-btn"
+      class={`${corner} crop-btn`}
```

Comment:
to continue with BEM, the `corner` could be considered a modifier:

`editor__crop-btn editor__crop-btn--${corner}`